index.html

```diff
@@ -5,7 +5,7 @@
     <title>Microverse Survey Form</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="css/style.css">
   </head>
 
```

Comment:
You can use semantic tag `header`, instead of using the `div` block with the '_header_' class. Also, you can use `main` tag instead of `div` block with '_wrapper_' class.  
Because, by using **semantic markup**, you help the browser understand the meaning of the content, define the purpose of the element. It also helps search engines to read the page and find the required information faster.
So I highly recommend using semantic tags. You can find additional info [here](https://www.bitdegree.org/learn/html5-semantic-tags#:~:text=HTML5%20semantic%20tags%20define%20the%20purpose%20of%20the%20element.,find%20the%20required%20information%20faster.)